web/src/beta/features/PluginPlayground/Plugins/presets/ui/sidebar.ts

```diff
@@ -26,162 +25,127 @@ extensions:
-              <span class="menu-title">Home</span>
-              <span class="menu-dots">⋮</span>
-            </li>
-            <li>
-              <span class="menu-icon">ℹ️</span>
-              <span class="menu-title">About</span>
-              <span class="menu-dots">⋮</span>
-            </li>
-            <li>
-              <span class="menu-icon">📞</span>
-              <span class="menu-title">Contact</span>
-              <span class="menu-dots">⋮</span>
-            </li>
-            <li>
-              <span class="menu-icon">❓</span>
-              <span class="menu-title">FAQ</span>
-              <span class="menu-dots">⋮</span>
-            </li>
-            <li>
-              <span class="menu-icon">💡</span>
-              <span class="menu-title">Help</span>
-              <span class="menu-dots">⋮</span>
-            </li>
-            <li>
-              <span class="menu-icon">⚙️</span>
-              <span class="menu-title">Settings</span>
-              <span class="menu-dots">⋮</span>
-            </li>
-            <li>
-              <span class="menu-icon">👤</span>
-              <span class="menu-title">Profile</span>
-              <span class="menu-dots">⋮</span>
-            </li>
-            <li>
-              <span class="menu-icon">📊</span>
-              <span class="menu-title">Dashboard</span>
-              <span class="menu-dots">⋮</span>
-            </li>
-            <li>
-              <span class="menu-icon">🔔</span>
-              <span class="menu-title">Notifications</span>
-              <span class="menu-dots">⋮</span>
-            </li>
-            <li>
-              <span class="menu-icon">📄</span>
-              <span class="menu-title">Reports</span>
-              <span class="menu-dots">⋮</span>
-            </li>
-
-          </ul>
-        </div>
+  sourceCode: `// A collapsible sidebar navigation panel with menu items.
+
+  reearth.ui.show(\`
+  <style>
+    /* Generic styling system that provides consistent UI components and styling across all plugins */
+
+    @import url("https://reearth.github.io/visualizer-plugin-sample-data/public/css/preset-ui.css");
+
+  /* Plugin-specific styling */
+
+    .content-wrapper {
+      overflow: hidden;
+      height: calc(100vh - 20px);
+      display: flex;
+      flex-direction: column;
+    }
+
+    .upside {
+      position: sticky;
+      top: 0;
+      left: 0;
+      z-index: 10;
+      padding: 10px 0;
+    }
+
+    #toggleBtn {
+      background: none;
+      border: none;
+      cursor: pointer;
+      font-size: 24px;
+      color: #555;
+    }
+
+    .menu-container {
+      flex-grow: 1;
+      overflow-y: auto;
+      margin-top: 10px;
+    }
+
+    .menu li .menu-icon {
+      margin-right: 10px;
+    }
+
+    .menu li .menu-title {
+      flex-grow: 1;
+    }
+
+    .menu li .menu-dots {
+      font-size: 18px;
+      color: #999;
+    }
+
+    .hidden {
+      visibility: hidden;
+      opacity: 0;
+    }
```

Comment:
Should be in preset.
Base on the style you gave, the name might be better as `invisible`.